dbt_project.yml

```diff
@@ -69,5 +72,8 @@ vars:
   marts_prefixes: ['fct_', 'dim_']
   other_prefixes: ['rpt_']
 
+  # -- Performance variables --
```

Comment:
@dave-connors-3 @b-per I'm curious if either of you have any thoughts as to what this threshold should be / if this is the proper way to be testing this model?

Reply:
This is different than the way `models_fanout_threshold` is handled - but my assumption was that in this case, someone might want to see all models dependent on chains of views when querying this model (rather than just the ones that surpass their threshold). Let me know what you think

Reply:
I like to show all of them and then just have the test on the `fct` failing, like you did.